.github/workflows/vfd-ros3.yml

```diff
@@ -0,0 +1,448 @@
+name: Build and test HDF5 ROS3 VFD
+
+on:
+  workflow_call:
+    inputs:
+      build_mode:
+        description: "CMake build type (CMAKE_BUILD_TYPE)"
+        required: true
+        type: string
+      aws_c_s3_build_type:
+        description: "Install aws-c-s3 from a package manager ('package') or build from source ('source')"
+        required: true
+        type: string
+      aws_c_s3_tag:
```

Comment:
should add a default value

Reply:
I'm actually checking for empty input and letting the checkout action use the default branch if so. Probably a bit more future proof than specifying a default (though let's hope the name of the default branch doesn't change in GH again).